src/components/Finder.js

```diff
@@ -80,9 +86,22 @@ const useStyles = makeStyles((theme) => ({
       borderBottom: `2px solid ${theme.palette.primary.main}`,
     },
   },
-  iconTrustActice: {
-    '& path': {
-      fill: 'inherit',
+  tabNavigationMain: {
```

Comment:
This styling has to do with tab navigation (tab navigation action) and should be in that component. The styling is missing in activity log